dask_geomodeling/tests/test_core.py

```diff
@@ -169,9 +169,11 @@ def test_cache_token(self, patched_tokenize):
         """A Block's token is saved"""
         block = MockBlock(1)
         _ = block.token
-        patched_tokenize.assert_called_once()  # tokenized 1 argument
+        # tokenized 1 argument
+        self.assertEqual(1, patched_tokenize.call_count)
```

Comment:
Waarom is deze wijziging nodig?

Reply:
`assert_called_once` bestaat niet in python 3.5

Reply:
Ah, je gebruikt de mock uit python zelf. Ik ging er van uit dat het een eigenschap van de staat-op-pypi-mock-library was en dat het dus onafhankelijk is.

(Als het pytest is kan je ook `assert 1 == patched_tokenize.call_count` doen.)